tests/anonymous-credentials/schema.spec.ts

```diff
@@ -349,7 +582,7 @@ describe('CredentialBuilder Schema', () => {
   it('subject as an array', () => {
     const schema6 = getExampleSchema(6);
     const cs6 = new CredentialSchema(schema6);
-    expect(cs6.schema[SUBJECT_STR]).toEqual(schema6[SUBJECT_STR]);
+    expect(cs6.jsonSchema.properties[SUBJECT_STR]).toEqual(schema6.properties[SUBJECT_STR]);
```

Comment:
this is an awful way of accessing the schema object, having var.jsonSchema.properties instead of just var.properties

Reply:
What is awful about that? `properties` no longer exists because now there are 2 schema objects and `properties` name alone can be confusing. But what is awful about that exactly?

Reply:
touched on it in another comment but a library user may believe that they can modify the jsonSchema attribute to make changes, which afaik wont be the case as the constructor is parsing it into an internal representation. i think itd be better that its private and not intended for the user to access it

Reply:
If the only concern is unintentional changes, then it applies to a bunch of other properties across classes and we can guard that using readonly keyword.  I added that in [this commit](https://github.com/docknetwork/crypto-wasm-ts/pull/5/commits/f53bde90c48d8b2019467bf210b7270d8a3bb00a)